VariableAndDataType/data_type.md

```diff
@@ -1,32 +1,52 @@
 ## Data Type
 
-Variables in JavaScript can hold various types of data such as strings, numbers, booleans, objects, or even functions. 
+Variables in JavaScript can hold various types of data such as strings, numbers, booleans, objects, or even functions.
 
-**Data types** define the types of values that can be used in the language. JavaScript has several data types:
+**Data types** define the types of values that can be used in the language. JavaScript has several data types that can be broadly categorized into two groups: primitive data types and reference data types.
 
-- Primitive Data Types:
-Number: Represents both integer and floating-point numbers. Example: 42, 3.14
+### Primitive Data Types:
 
-    - String: Represents sequences of characters, enclosed in either single quotes (') or double quotes ("). Example: 'Hello', "JavaScript"
+in programming languages are called "primitive" because they are fundamental, basic building blocks upon which more complex data types and structures are built. They are not composed of smaller parts and are not defined in terms of other data types.
```

Comment:
capital letter

Reply:
Good eyes, I missed that. I capitalized the beginning of the phrase.